File: Data Structures & Algorithms/two-integer-sum/submission-3.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
    //    int i = 0;
    //    int j = nums.size()-1;

    //    while( i <= j ) {
    //     int sum = nums[i] + nums[j];
    //     if( sum == target ) {
    //         return{i,j};
    //     }
    //     else if( sum < target ) {
    //         i++;
    //     }
    //     else 
    //     j--;
    //    }
    //    return {-1,-1};
    for( int i=0; i<nums.size(); i++ ) {
        int sum = 0;
        for( int j=i+1; j<nums.size(); j++ ) {
            sum = nums[i] + nums[j];
            if( sum == target ) {
                return{i,j};
            }
        }
    }
    return {-1,-1};
    }
};

File: Data Structures & Algorithms/remove-element/submission-3.java
class Solution {
    public int removeElement(int[] nums, int val) {
        
        int ct =0;
        int j = 0;
        for( int i=0; i<nums.length; i++ ) {
            if( nums[i] != val  ) {
                ct++;
                nums[j++] = nums[i];
            }
        }
        return ct;
    }
}